global-shared-library/src/main/resources/archetype-resources/shared-library/vars/acme.groovy

```diff
@@ -0,0 +1,13 @@
+// vars/acme.groovy
+class acme implements Serializable {
```

Comment:
BTW I do not actually recommend ever declaring types in `vars/*.groovy`. Should just delete the part of the docs that discusses it.

Reply:
Then it is better to delete it from here.